### PR TITLE
Added custom sidebar template

### PR DESCRIPTION
**Pull request by ria-arora**

### CODE REVIEW

File: customSidebarTemplate/sidebar.gs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* Sidebars do not suspend the server-side script while the dialog is open.
+*/
+function showSidebar() {
+  // Creates a new HtmlOutput object from a file in the code editor.
+  var html = HtmlService.createHtmlOutputFromFile('SidebarPage') 
+      .setTitle('Present Sidebar')
+      .setWidth(300);
+  SpreadsheetApp.getUi() 
+      .showSidebar(html); // Opens sidebar in the user's editor with custom client-side content
+}
```
Comment (paneendra-google) [inline customSidebarTemplate/sidebar.gs:28]: Can we also add a function that opens the same form using showModalDialog?

**Review by paneendra-google — CHANGES_REQUESTED** [review]

File: customSidebarTemplate/sidebar.gs (hunk 1 above)
Comment (ria-arora) [reply customSidebarTemplate/sidebar.gs:28]: Sure, I will add it!

File: customSidebarTemplate/SidebarPage.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      function sendText(){
+        // The getElementById() method returns the element that has the ID attribute with the specified value.
+        var inputText = document.getElementById("input-text").value;
+        // google.script.run is an asynchronous client-side JavaScript API
+        // that can call server-side Apps Script functions.
+        google.script.run.textFunction(inputText);
+      }
+      // Send selected list element value to the relevant server-side Apps Script function
+      function sendListElement(){
+        var selectedListElement = document.getElementById("dropdown-list").value
```
Comment (gargabhay) [inline customSidebarTemplate/SidebarPage.html:116]: nit: semi colon

Comment (gargabhay) [inline customSidebarTemplate/SidebarPage.html]: nit: semi colon

File: customSidebarTemplate/sidebar.gs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+function radioButtonFunction(selectedRadioButton) {
+  // Add functionality to be performed with the selected radio button value
+}
+
+function checkBoxFunction(selectedCheckboxes) {
+  // Add functionality to be performed with the array of selected checkboxes 
+}
+
+
```
Comment (gargabhay) [inline customSidebarTemplate/sidebar.gs]: remove

**Review by gargabhay — APPROVED** [review]

**Review by paneendra-google — APPROVED** [review]

File: customSidebarTemplate/SidebarPage.html (hunk 2 above)
Comment (ria-arora) [reply customSidebarTemplate/SidebarPage.html]: Thank you, made the changes

**Pull request merged**